### PR TITLE
updated already submitted url

### PR DESCRIPTION
**Pull request by tchow8**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

When the user attempts to start an application (based on their IDAM account) but they already have a submitted application, then it redirects them to the /error-application-submitted page.
This is not user friendly as it implies there is an error, so the url has been requested to be changed to /application-submitted.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

Existing tests have been modified to match this change

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/steps/error-submitted/index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const runStepHandler = require('app/core/handler/runStepHandler');
 
 module.exports = class SubmittedError extends Page {
```
Comment (Rob-Doherty) [inline app/steps/error-submitted/index.js]: Is this still an `Error` class? If not should the class and folder name change as well?

Comment (tchow8) [reply app/steps/error-submitted/index.js]: Not an Error class, just named as such it seems. I'll rename if you want.

Comment (Rob-Doherty) [reply app/steps/error-submitted/index.js]: that would be good for clarification, otherwise it could get confusing in a future refactor

**Comment by tchow8** [comment]
Moved to PR #27

**Pull request closed without merging**